### DISCUSSION TITLE
[@types/node] stream/web.d.ts lacks DecompressionStream

### DISCUSSION THREAD

**Discussion by wydengyre** [DefinitelyTyped/DefinitelyTyped#68060] [Issues with a @types package] [answered]
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/node/stream/web.d.ts

This file should define types for the `DecompressionStream` class: https://nodejs.org/api/webstreams.html#class-decompressionstream

[a comment by an automated account is not shown]

**Comment by ghost** [comment, marked as answer]
Added in #68214
